src/satosa/frontends/saml2.py

```diff
@@ -726,11 +727,25 @@ def handle_authn_response(self, context, internal_response):
         :return:
         """
 
+        return self._handle_authn_response(context, internal_response)
+
+    def _handle_authn_response(self, context, internal_response):
+        """
+        """
         # Using the context of the current request and saved state from the
-        # authentication request dynamically create an IdP instance and then
-        # use it to handle the authentication response.
+        # authentication request dynamically create an IdP instance.
         idp = self._create_co_virtual_idp(context)
-        return self._handle_authn_response(context, internal_response, idp)
+
+        # Add any static attributes for the CO.
+        co_config = self._get_co_config(context)
+
+        if self.KEY_CO_ATTRIBUTES in co_config:
+            attributes = internal_response.attributes
+            for attribute, value in co_config[self.KEY_CO_ATTRIBUTES].items():
+                attributes[attribute] = value
```

Comment:
Internal-attribute values are of type `list`; this is because some attributes can have multiple values.
If you want to keep the same interface, I'd make this 

```
from typing import Iterable

[...]

attributes[attribute] = (
    value                          # keep the value as is
    if isinstance(value, Iterable) # if it is already a list (like noreduorgacronym)
    and not isinstance(value, str) # and not a string (strings are iterables)
    else [value]                   # otherwise wrap it in a list
)
```